### ISSUE TITLE
error on timestamp round

### ISSUE THREAD

**Issue by mmeloni** [codenotary/immudb#1044]
**What happened**
After retrieving row it's seems there is a problem after time round
`expect: 2021-12-17 12:28:35.975274132 +0100 CET, got -62032-01-18 09:05:24.038176 +0000 UTC after time round`

Involved query:
`SELECT * FROM users WHERE id = ? AND users.deleted_at IS NULL ORDER BY users.id LIMIT 1`

**What you expected to happen**

**How to reproduce it (as minimally and precisely as possible)**

**Environment**
```shell
# run "immu* version" and copy/paste the output here
```

**Additional info (any other context about the problem)**
Run this https://github.com/go-gorm/gorm/blob/master/tests/associations_belongs_to_test.go#L16 on immugorm

**Comment by tauu** [comment]
PR #1045 describes what causes the bug and provides a fix for it. :-)

Side note: is immugorm publicly available? I am somehow not able to find it. 😅

**Comment by mmeloni** [comment]
Hey @tauu, thanks very much. Stay tuned ,next days there will be surprises

**Comment by tauu** [comment]
Something like a Christmas surprise? Looking forward to that. ;-) 

@mmeloni Maybe you are the right person to ask about that: I have created a separate database/sql driver for immudb: https://github.com/tauu/immusql . Unfortunately I only found out that there is an official one in development when I was reading the release notes of 1.2.0. In contrast to the stdlib driver this one also supports opening a database on the local filesystem by creating an embedded engine for it. The main idea for this is to simplify unit testing for programs using immudb. Tests could just set a dsn for a local database and don't have to worry about spinning up a database service first.

Is this be a feature that would be of interest for the stdlib driver? In that is the case I can happily create a PR for it.
